src/main/java/edu/ucsb/cs156/happiercows/entities/CowLot.java

```diff
@@ -0,0 +1,29 @@
+package edu.ucsb.cs156.happiercows.entities;
+
+import javax.persistence.*;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "cowlots")
+
+public class CowLot {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name="commons_id")
+    private long commonsId;  
+
+    @Column(name="user_id")
+    private long userId;  

```

Comment:
or you could just use the userCommonsId

I think that will end up being easier down the road.  You'll only have to deal with one key instead of two, and I think in every circumstance where you'd be buying cows, you have access to a userCommons object.